.gitignore

```diff
@@ -0,0 +1,3 @@
+package-lock.json
+.DS_Store
+./templates/.DS_Store
```

Comment:
Can you change this so all .DS_Store files are excluded, regardless of the directory they are in?